README.md

```diff
@@ -335,6 +322,29 @@ tokenName: 'token',
 tokenRoot: false,
 
 
+// Refresh Token Options
+// =====================
+
+// Option to turn refresh tokens On/Off
+useRefreshToken: false,
+// The option to enable/disable the automatic refresh of Auth tokens using Refresh Tokens
+autoUpdateToken: true,
+// The the property from which to get the refresh token after a successful token refresh
+refreshTokenProp: 'refresh_token',
+// Prepended to the `refreshTokenName` when kept in storage (nothing to do with refresh token responses)
+refreshTokenPrefix: 'aurelia',
+// Oauth Client Id
+clientId: false,
+
+// If the property defined by `refreshTokenProp` is an object:
+// ------------------------------------------------------------
+
+// This is the property from which to get the token `{ "refreshTokenProp": { "refreshTokenName" : '...' } }`
+refreshTokenName: 'token',
```

Comment:
How about we remove the token **prefix** altogether and replace it with something (e.g.  `refreshTokenStorage`) that specifies the whole storage key? This whole prefix using another unrelated option to create the storage key kinda sucks, don't you think?